extensions/eclipse-che-theia-plugin-ext/src/browser/style/che-plugins-caching.css

```diff
@@ -39,5 +39,5 @@
 }
 
 .che-plugins-caching-progress-status b {
-    color: var(--theia-success-color2);
+    color: var(--theia-successBackground);
```

Comment:
color property sets a text color. Could you pick another variable, not  --theia-successBackground ?

Reply:
updated to `--theia-badge-foreground` which is best suited for that